x-pack/plugin/sql/jdbc/src/test/java/org/elasticsearch/xpack/sql/jdbc/JdbcConfigurationTests.java

```diff
@@ -75,6 +75,23 @@ public void testDebugOut() throws Exception {
         assertThat(ci.debugOut(), is("jdbc.out"));
     }
 
+    public void testDebugBuffered() throws Exception {
+        JdbcConfiguration ci = ci("jdbc:es://a:1/?debug=true&debug.buffered=false");
```

Comment:
I think `debug.flushAlways` or `debug.noBuffer` is more meaningful. I also like the consistency of the options - by default they are false and need to be turn on through true.
That is the default (`false`) is to always buffer but true would yield an immediate flush.